README.md

```diff
@@ -1583,7 +1583,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 | [Back To Top](#contents) |
 
 - ### **S**
-
+  - [Subhranshu Patra](https://github.com/Subhranshu07)
```

Comment:
Please add your name in line 1831, after Subhrajit Dutta.
Let us know if you need any help with that. 🙂

Reply:
Yes it's done now